server/api/models/question.js

```diff
@@ -14,10 +15,11 @@ const QuestionSchema = mongoose.Schema({
       },
       is_correct: {
         type: Boolean,
-        default: false
+        default: false,
+        required: true
```

Comment:
Does it make sense to mark `is_correct` as required field, as it is always set as `false` by default if we don't provide any value?

Reply:
@skyshader Oh yes, you're right. I will make the changes

Reply:
@skyshader Actually I wanted to make `value` as `required`. Sorry for the mistake.